src/Provider/TodoProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Fakturoid\Provider;
+
+use Fakturoid\DispatcherInterface;
+use Fakturoid\Response;
+
+final class TodoProvider extends Provider
+{
+    public function __construct(
+        private readonly DispatcherInterface $dispatcher
+    ) {
+    }
+
+    /**
+     * @param array{'since'?:string, 'page'?:int} $params
+     */
+    public function list(array $params = []): Response
+    {
+        return $this->dispatcher->get(
+            '/accounts/{accountSlug}/todos.json',
+            $this->filterOptions($params, ['since', 'page'])
+        );
+    }
+
+    public function getToggleCompletion(int $id): Response
+    {
+        return $this->dispatcher->get(
+            sprintf(
+                '/accounts/{accountSlug}/todos/%d/toggle_completion.json',
+                $id
+            )
+        );
+    }
```

Comment:
Tady jsme měli chybu v dokumentaci, místo POST jsme tam měli GET, pardon. Můžeme to změnit na tohle?

```suggestion
    public function toggleCompletion(int $id): Response
    {
        return $this->dispatcher->post(
            sprintf(
                '/accounts/{accountSlug}/todos/%d/toggle_completion.json',
                $id
            )
        );
    }
```